### PR TITLE
Update local dev doc to remove requirement of creating a service account in GCP

### PR DESCRIPTION
**Pull request by keyonghan**
This is part of https://github.com/flutter/flutter/issues/58619

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,10 +116,10 @@ these values should be.
 
 If you see `Serving requests at 0.0.0.0:8080` the dev server is working.
 
-You may need to [create a service account in the GCP
-dashboard](https://pantheon.corp.google.com/iam-admin/serviceaccounts?project=flutter-dashboard&supportedpurview=project)
-to develop and test some features; these cannot be tested if you do
-not work for Google currently.
+To develop and test some features, you need to have a local `key.json` 
+copy of a service account in the GCP dashboard. Please refer to the 
+internal team doc for how to get a local copy; these cannot be tested if
```
Comment (digiter) [inline README.md:121]: Nit: provide a link to the internal team doc.

File: README.md [hunk 2]
```diff
@@ -116,10 +116,10 @@ these values should be.
 
 If you see `Serving requests at 0.0.0.0:8080` the dev server is working.
 
-You may need to [create a service account in the GCP
-dashboard](https://pantheon.corp.google.com/iam-admin/serviceaccounts?project=flutter-dashboard&supportedpurview=project)
-to develop and test some features; these cannot be tested if you do
-not work for Google currently.
+To develop and test some features, you need to have a local `key.json` 
+copy of a service account in the GCP dashboard. Please refer to the
```
Comment (digiter) [inline README.md:120]: If there's a dedicated service account, provide a link to it.

File: README.md (hunk 1 above)
Comment (keyonghan) [reply README.md:121]: is it okay to provide the internal doc link here?

File: README.md (hunk 2 above)
Comment (keyonghan) [reply README.md]: added.

File: README.md (hunk 1 above)
Comment (digiter) [reply README.md:121]: Yes, as long as it doesn't contain a confidential word.

Comment (keyonghan) [reply README.md]: Thanks. Updated.

File: README.md [hunk 3]
```diff
@@ -116,10 +116,10 @@ these values should be.
 
 If you see `Serving requests at 0.0.0.0:8080` the dev server is working.
 
-You may need to [create a service account in the GCP
-dashboard](https://pantheon.corp.google.com/iam-admin/serviceaccounts?project=flutter-dashboard&supportedpurview=project)
-to develop and test some features; these cannot be tested if you do
-not work for Google currently.
+To develop and test some features, you need to have a local `key.json` 
+copy of a service account ([dart-cocoon](https://pantheon.corp.google.com/iam-admin/serviceaccounts/details/110857733037195084916?project=flutter-dashboard)) in the GCP dashboard. Please refer to the
```
Comment (godofredoc) [inline README.md:120]: To develop and test some features, you need to have a local service account(`key.json`)
with access to the project you will be connecting to.

If you work for Google you can use the ... key with flutter-dashboard project.

Comment (keyonghan) [reply README.md]: updated.

**Pull request merged**